src/emulator/audio/include/audio/state.h

```diff
@@ -31,28 +30,19 @@
 typedef std::shared_ptr<SDL_AudioStream> AudioStreamPtr;
 typedef std::function<void(SceUID)> ResumeAudioThread;
 
-struct AudioOutput {
-    const uint8_t *buf = nullptr;
-    int len_bytes = 0;
-    SceUID thread = -1;
-};
-
 struct ReadOnlyAudioOutPortState {
     int len_bytes = 0;
-};
-
-struct AudioCallbackOutPortState {
-    AudioStreamPtr stream;
+    int freq = 0;
```

Comment:
Is this used anywhere?

Reply:
No, thanks for noticing it! I put it in when generating sine waves for debugging, and forgot to take it out.